FreeRTOS/FreeRTOS
# Replace absolute path with project relative path

Description
-----------
The debug launch configuration included with the CORTEX_A9_Zynq_ZC702_Vitis_QEMU project contained an absolute path.  This PR changes it to a project relative path.

Test Steps
-----------
Open the CORTEX_A9_Zynq_ZC702_Vitis_QEMU project in Vitis and inspect the launch configuration in the IDE.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
